src/waveform/vsyncthread.cpp

```diff
@@ -124,17 +116,8 @@ void VSyncThread::swapGl(QGLWidget* glw, int index) {
     glw->swapBuffers();
 #elif defined(__WINDOWS__)
     glw->swapBuffers();
-#else
-#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
-#ifdef QT_OPENGL_ES_2
-    glw->swapBuffers();
-#else
-    const QX11Info *xinfo = qt_x11Info(glw);
-    glXSwapBuffers(xinfo->display(), glw->winId());
-#endif
 #else
     glw->swapBuffers();
```

Comment:
This melds down to one glw->swapBuffers();
It is also part of https://github.com/mixxxdj/mixxx/pull/2088 waiting for merge. 
Can we merge this merge it first and revert this here?